src/patternfly/components/Alert/alert.scss

```diff
@@ -83,14 +91,26 @@
   background-color: var(--pf-c-alert--BackgroundColor);
   box-shadow: var(--pf-c-alert--BoxShadow);
 
+  // left border treatment base
+  // color is transparent by default, then set by the state modifier classes and turned on by the inline modifier
+  &::before {
```

Comment:
What do you think about moving this into the `&.pf-m-inline` selector since it isn't needed on the default alert?